views/components/info/control.es

```diff
@@ -36,6 +38,7 @@ config.on('config.set', (path, value) => {
 
 const PoiControl = connect((state, props) => ({
   muted: get(state, 'config.poi.content.muted', false),
+  tbtriggered: get(state, 'config.poi.touchbar.triggered', null),
```

Comment:
Then it will require an extra reducer. Use the UI store is better, but is it really necessary to do so?